### PR TITLE
Add favorite button

### PR DESCRIPTION
**Pull request by DanielN88**
# Description

Add favorite button

## Issues

Closes #

## Type of change

- [ ] Bug fix (non-breaking change which fixes an issue)
- [x] New feature (non-breaking change which adds functionality)
- [ ] Refactor feature (non-breaking change that modifies existing work)
- [ ] Cosmetic changes (changes to layout or visuals in CSS file)

## How Has This Been Tested?

- [x] console.log()
- [x] dev tools
- [x] functionality check through web browser
- [ ] tested by viewing the markdown file preview
- [ ] tested with testing framework (Cypress)

## Checklist:

- [x] My code follows the style guidelines of this project (semicolons, indentation, naming conventions, etc..)
- [x] I have performed a self-review of my code
- [ ] I have commented my code, particularly in hard-to-understand areas
- [x] I have deleted all comments/console.log() after functionality and understanding is final
- [ ] I have added any issues closed by this PR in the Issues section.
- [ ] I have added others as reviewers

### CODE REVIEW

File: src/Components/Description.js [hunk 1]
```diff
@@ -1,17 +1,26 @@
...
+    dispatch(favorite(currentArt));
+  };
 
   return (
     <div className="image-description">
       <p className="art-title"><strong>Title:</strong> {currentArt.title}</p>
       <p className="art-artist"><strong>Artist: </strong>{currentArt.artist}</p>
       <p className="art-medium"><strong>Medium: </strong>{currentArt.medium}</p>
       <p className="art-period"><strong>Period: </strong>{currentArt.century}</p>
+      <button onClick={() => addFavorite(currentArt)}>Add to favorites</button>
```
Comment (whiteheadol) [inline src/Components/Description.js:23]: I'm so happy we figured this out!

Comment (Jlavarine) [inline src/Components/Description.js:23]: A great way to add favorites! Now to display that data👍

File: src/features/favorites/favoriteArtSlice.js [hunk 2]
```diff
@@ -0,0 +1,14 @@
+import { createSlice, createAsyncThunk } from '@reduxjs/toolkit'
```
Comment (whiteheadol) [inline src/features/favorites/favoriteArtSlice.js:1]: I can't believe this worked so quickly!

**Review by Jlavarine — APPROVED** [review]
Awesome! This app is gaining functionality

**Review by whiteheadol — APPROVED** [review]

**Pull request merged**